Assets/Scripts/DataCollection/PlayerDataRecorder.cs

```diff
@@ -20,6 +20,10 @@ public class PlayerDataRecorder : MonoBehaviour
     
     void Start()
     {
+        if(!GetComponent<PlayerManager>().IsPlayerLocal())
+        {
```

Comment:
PlayerDataRecorder is assigned to the player object right? So you should be able to just use `photonView.IsMine` here

Reply:
So this would be `this.enabled = photonView.IsMine`

Reply:
You can use same form even if `photonView.IsMine` is not valid to use here

Reply:
This script does not have access to photonView. I think we have to inplement one of the interfaces for that which will just make things even more messy.

Reply:
oke